### PR TITLE
Override jobs

### LINKED ISSUE #561: Jobs override v2
Jobs should be overridden in a manner such that:

## The structure of assets.json is something like this:
```
{
  "assets": {
    "collection": {
      "ethCollection": {
        "power": 2,
        "official jobs": {
          "1": {
            "URL": "http://kucoin.com/eth1",
            "selector": "eth1",
            "power": 2,
            "weight": 2
          },
          "2": {
            "URL": "http://api.coinbase.com/eth2",
            "selector": "eth2",
            "power": 3,
            "weight": 2
          }
        },
        "custom jobs": [
          {
            "URL": "http://127.0.0.1/eth1",
            "selector": "eth1",
            "power": 2,
            "weight": 3
          },
          {
            "URL": "http://127.0.0.1/eth2",
            "selector": "eth2",
            "power": 2,
            "weight": 2
          },
          {
            "URL": "http://127.0.0.1/eth3",
            "selector": "eth3",
            "power": 2,
            "weight": 2
          }
        ]
      }
    }
  }
}
```
The custom jobs array will contain all the customs jobs that have to be overridden and razor-go will pick these jobs for the collection.

- [ ] jobs.json renamed to assets.json
- [ ] Check assets.json for a collection, if that collection is present, pull all the data from the official jobs and custom jobs (if it exists)
- [ ] If the power of the collection is mentioned in assets.json, use that power value.
- [ ] Aggregation method however will always be fetched from the smart contract.
- [ ] The override flag will be removed and override will be implicit.

### PR DESCRIPTION
**Pull request by Yashk767**
# Description

Added override jobs functionality and fetching custom jobs data from assets.json file.

Fixes #561

### CODE REVIEW

File: utils/asset.go [hunk 1]
```diff
@@ -229,18 +232,59 @@ func (*UtilsStruct) GetActiveAssetsData(client *ethclient.Client, epoch uint32)
...
+
+		// Also adding custom jobs to jobs array
+		customJobs := GetCustomJobsFromJSONFile(collection.Name, dataString)
+		jobs = append(jobs, customJobs...)
+	}
+
+	for _, id := range collection.JobIDs {
+
+		// Ignoring the Jobs which are already overriden and added to jobs array
+		if !Contains(overriddenJobIds, jobs) {
```
Comment (ashish10677) [inline utils/asset.go:274]: Change `jobs` to `id`

**Review by ashish10677 — CHANGES_REQUESTED** [review]
Remove OverrideJobs command.

**Comment by ashish10677** [comment]
@Yashk767 please update the readme as well.

**Comment by Yashk767** [comment]
> @Yashk767 please update the readme as well.

Done

File: README.md [hunk 2]
```diff
@@ -752,6 +701,94 @@ docker run -it  --network razor_network\
     razornetwork/razor-go:latest \
     setConfig --exposeMetrics 2112
 ```
+
+### Override Job and Adding Your Custom Jobs
+
+Jobs URLs are a placeholder for where to fetch values from. There is a chance that these URLs might either fail, or get razor nodes blacklisted, etc.
```
Comment (ashish10677) [inline README.md:707]: for should be from.

File: README.md [hunk 3]
```diff
@@ -752,6 +701,94 @@ docker run -it  --network razor_network\
...
+"official jobs": {
+          "1": {
+            "URL": "https://data.messari.io/api/v1/assets/eth/metrics",
+            "selector": "[`data`][`market_data`][`price_usd`]",
+            "power": 2,
+            "weight": 2
+          },
+```
+
+- Additional jobs tha you want to add to a collection should be added in `custom jobs` field with their respective URLs and selectors.
```
Comment (ashish10677) [inline README.md:753]: `tha` should be `that`

File: README.md [hunk 4]
```diff
@@ -752,6 +701,94 @@ docker run -it  --network razor_network\
...
+          {
+            "URL": "https://api.lunarcrush.com/v2?data=assets&symbol=ETH",
+            "selector": "[`data`][`0`][`price`]",
+            "power": 3,
+            "weight": 2
+          },
+        ]
+```
+
+### Delete override
```
Comment (ashish10677) [inline README.md:767]: Are we still supporting delete override @SkandaBhat ? I don't think so.

**Review by ashish10677 — CHANGES_REQUESTED** [review]

**Comment by Yashk767** [comment]
Removed `deleteOverride` command.

File: README.md (hunk 4 above)
Comment (ashish10677) [reply README.md:767]: @Yashk767 please remove this from README.

Comment (Yashk767) [reply README.md]: > @Yashk767 please remove this from README.

Done

**Review by ashish10677 — APPROVED** [review]

**Pull request merged**